cli/pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+requires = ["scikit-build-core>=0.3.3", "pybind11"]
+build-backend = "scikit_build_core.build"
+
+
+[project]
+name = "graphar_cli"
+version = "0.0.1"
+description = "GraphAr command line tool"
+readme = "README.md"
+authors = [{ name = "GraphAr community", email = "dev@graphar.apache.org" }]
+requires-python = ">=3.7"
+dependencies = ["typer ~= 0.12.3", "pydantic ~= 2.8.2", "pyyaml ~= 6.0.2"]
+
+[project.optional-dependencies]
+test = ["pandas ~= 2.2.2"]
+
+[project.scripts]
+graphar-cli = "graphar_cli.graphar_cli:main"
```

Comment:
Is a `graphar-cli` a good name? I would expect something like just `graphar`

Reply:
+1